README.md

```diff
@@ -57,9 +57,51 @@ We plan to have a very simple internal record of metrics with a limited data ret
 
 #### AWS CloudWatch
 
-TBA
+Sends metric data to [AWS Cloudwatch](https://docs.aws.amazon.com/cloudwatch/).
+
+##### Requires
+- [local/aws](https://github.com/catalyst/moodle-local_aws) plugin.
+- AWS account
+- IAM user.
+
+##### IAM Access
+
+You need to have an IAM user connected to your AWS account.
+This user will need permissions to submit cloudwatch data.
+
+This is the minimum user policy for access.
+
+    {
+        "Version": "2012-10-17",
+        "Statement": [{
+            "Effect": "Allow",
+            "Action": [
+                "cloudwatch:PutMetricData"
+            ],
+            "Resource": "*"
+        }]
+    }
+
+Or the user can be assigned the `CloudWatchFullAccess`
+predefined policy.
+
+Ideally, the execution environent will be configured with
+the IAM credentials. However, the following config can
+also be used. Substitute the value strings with your access keys.
+
+    $CFG->cltr_cloudwatch['credentials']['key'] = '<Access key ID>';
```

Comment:
Can you please convert this to use standard admin settings so they can *optionally* be in the UI, but can still be forced in config.php and update the docs here to be like

```
$CFG->forced_plugin_settings['cltr_cloudwatch']['credentials'] = 'blah';
```

Reply:
Use the password admin setting for secrets, it does some magic when the value is forced so you can't see it in the gui

Reply:
Do you want me to add to add settings for these?

Reply:
Yes

Reply:
example 

https://github.com/catalyst/moodle-search_elastic/blob/MOODLE_310_STABLE/settings.php#L55-L61